runtime/src/append_vec.rs

```diff
@@ -113,52 +113,29 @@ pub struct AppendVec {
     append_offset: Mutex<usize>,
     current_len: AtomicUsize,
     file_size: u64,
+    read_only: bool,
 }
 
 impl Drop for AppendVec {
     fn drop(&mut self) {
+        // the env knob is needed to make solana-validator retain retired AppendVecs for
+        // later inspection of the files
```

Comment:
@sakridge Does this comment now make sense to justify my laziness of another env var? ;)

Reply:
I'd vote for the following over an env var:
1. Considerable plumbing
2. A lazy_static like https://github.com/solana-labs/solana/blob/40737e9efa4834300df17383f4479fa90b5d8260/runtime/src/accounts_db.rs#L61-L65

When I added (2), I declined (1) myself :)

Reply:
hehe, thanks for the relevant info! I'll first try the (1). :)